internal/entimport/mysql.go

```diff
@@ -67,7 +81,7 @@ func (m *MySQL) field(column *schema.Column) (f ent.Field, err error) {
 	case *schema.TimeType:
 		f = field.Time(name)
 	default:
-		return nil, fmt.Errorf("entimport: unsupported type %q", typ)
+		return nil, fmt.Errorf("column %v: unsupported type %q", column.Name, typ)
```

Comment:
```suggestion
		return nil, fmt.Errorf("entimport: unsupported type %q for column %v", typ, column.Name)
```

Reply:
I pulled these because this gets wrapped higher in the stack.